libraries/AP_AccelCal/AccelCalibrator.cpp

```diff
@@ -379,9 +379,6 @@ void AccelCalibrator::run_fit(uint8_t max_iterations, float& fitness)
         }
 
         num_iterations++;
-        if (fitness - last_fitness < 1.0e-9f) {
-            break;
-        }
```

Comment:
humn... got it... I think the intention was to take an `abs(fitness - last_fitness)` to detect cases in which the value isn't changing.